src/controller/python/chip-device-ctrl.py

```diff
@@ -560,6 +560,74 @@ def do_discover(self, line):
 
         self.devCtrl.PrintDiscoveredDevices()
 
+    def do_ping(self, line):
+        """
+        ping [options] <nodeid>
+
+        Options:
+          -i  <interval>  ping interval time in seconds
```

Comment:
Why is it seconds here and milliseconds in the API?

Reply:
This is to align with chip-shell and test spec, we want to use the same test cases when migrating to python device controller.